pages/1.12/deploying-services/private-docker-registry/index.md

```diff
@@ -235,8 +235,45 @@ Follow these steps to add your Docker registry credentials to the [DC/OS Enterpr
 
    The Docker image will now pull using the provided security credentials.
 
+<a name="docker-repo-certs"></a>
+
+# Configuring the Docker registry to use a custom certificate
+As an alternative to providing user credentials to authorize access to the private Docker registry, you can use a custom certificate to authorize access to the private Docker registry.
```

Comment:
This isn't really an "alternative".   Some registries require BOTH credentials and a valid certificate to access, the certificate encrypts the communications between the client and the registry, while the credentials are used to determine who gets to access it.

So I would suggest something like "If your private registry uses a certificate to secure communications with it, follow the following steps to configure the agent nodes to trust the certificate used with your private registry".

Reply:
Modified.